static/include/app/apis/generated/app.md

```diff
@@ -1770,7 +1807,7 @@ Get registry item by ID.
 
 **Parameters:**
 
-- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item.
+- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item. This is the namespace and name of the item in the form `namespace:name`. For example, [Viam's `csi-cam-pi` module's](https://app.viam.com/module/viam/csi-cam-pi) item ID would be `viam:csi-cam-pi`.
```

Comment:
This is the case for public items, not for private ones. 
```suggestion
- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item. For public items, this is the namespace and name of the item in the form `namespace:name`. For example, [Viam's `csi-cam-pi` module's](https://app.viam.com/module/viam/csi-cam-pi) item ID would be `viam:csi-cam-pi`. For private items, it is `org-id:name`. For example, `abc01234-0123-4567-ab12-a11a00a2aa22:training-script`.
```

Reply:
oh didn't catch that, thank you!